pyoverkiz/client.py

```diff
@@ -519,6 +521,7 @@ async def fetch_events(self) -> list[Event]:
         Per-session rate-limit : 1 calls per 1 SECONDS period for this particular
         operation (polling)
         """
+        await self._refresh_token_if_expired()
```

Comment:
Why is this required here? Since we call post a line later and in post we also call this function?

Reply:
Because we want to refresh the `event_listener_id` before fetching the events. That’s the point of this PR.
If we don’t do that, we will call the `__post` with an expired token, and so an invalid `event_listener_id`. Even if we refresh the token into the `__post` method, the `event_listener_id` which is a parameter will still be wrong.